### PR TITLE
Removes 12 non-scientist Xenoarch Spawns.

### PR DESCRIPTION
**Pull request by Owai-Seek**
## About The Pull Request

Removes the following spawns from Xenoarch:
2x Medical Doctor
2x Paramedic
1x Chemist
2x Assistant
2x Atmos Tech
2x Engineering
1x Science Security Officer

## Why It's Good For The Game

It's confusing to new players, and there's like, a 50% chance than all of medical will spawn on xenoarch roundstart.

## Changelog
:cl:
tweak: Removed all roundstart non-scientist xenoarch spawns.
/:cl:

### CODE REVIEW

**Comment by Owai-Seek** [comment]
Those commits are evidence of my past failures to keep my master clean.

**Comment by jjpark-kb** [comment]
no, there is already my PR.

**Comment by Owai-Seek** [comment]
I was told xenoarch wasn't causing the crashing, so I assumed that PR wouldnt go through.

File: _maps/map_files/Mining/Lavaland_Skyrat.dmm [hunk 1]
```diff
@@ -698,17 +698,6 @@
 	},
 /turf/open/floor/plasteel,
 /area/xenoarch/eng)
-"dF" = (
```
Comment (Gandalf2k15) [inline _maps/map_files/Mining/Lavaland_Skyrat.dmm:701]: Oye! You've deleted things you shouldn't have here!

File: _maps/map_files/Mining/Lavaland_Skyrat.dmm [hunk 2]
```diff
@@ -9015,16 +8990,6 @@
 	},
 /turf/open/floor/plasteel,
 /area/xenoarch/arch)
-"Us" = (
```
Comment (Gandalf2k15) [inline _maps/map_files/Mining/Lavaland_Skyrat.dmm:9018]: And here!

**Comment by Owai-Seek** [comment]
When you check the map diffs it only shows the spawns being removed. I made sure when editing to not use hotkeys to delete stuff, and intentionally right-click deleted each spawn instead of hotkey deleting them.

I made the PR so I could get the map diffs to make sure it wasn't being fucky.

**Comment by jjpark-kb** [comment]
This will be dealt with later.

**Pull request closed without merging**